### ISSUE TITLE
Issue with deploy the coolqlcool with Now 

### ISSUE THREAD

**Issue by ibrahimBelkhiria** [dinubs/coolqlcool#6]
Hi , I can not deploy the project with Now 
I followed the steps : 
 1/ clone the project 
2/ run npm install 
3/ run the command line now (after I setup my zeit account) 
The application is deployed but the url is showing the index.js code !! 

Please how can  I fix this ?

**Comment by alee792** [comment]
The one click deployment also seems to be failing, I'm guessing because there's now now.json present?

**Comment by ibrahimBelkhiria** [comment]
@alee792  Did you find a solution ?

**Comment by alee792** [comment]
Nope, I think we'll just have to wait for a now.json, or you can write your own deployment.

**Comment by dinubs** [comment]
hmm... I'll take a look over the weekend, this was working the last time I checked it. Thanks for reporting :D

**Comment by ibrahimBelkhiria** [comment]
@dinubs Thank you for your efforts  :)

**Comment by dinubs** [comment]
@ibrahimBelkhiria Do you have a URL that I can use to see that this is happening, I'm having difficulty reproducing the initial issue with you only seeing `index.js`.

**Comment by dinubs** [comment]
In regards to the one-click deploy it seems to be broken for a lot of people. https://github.com/zpnk/deploy.now/issues/29

**Comment by ibrahimBelkhiria** [comment]
@dinubs  Yes of course        https://coolqlcool-master-jq9eo2084.now.sh/
